src/command/cmd_funcs.c

```diff
@@ -9286,3 +9292,91 @@ cmd_mam(ProfWin* window, const char* const command, gchar** args)
 
     return TRUE;
 }
+
+gboolean
+cmd_editor(ProfWin* window, const char* const command, gchar** args)
+{
+    /* Build temp file name */
+    GString * tempfile = g_string_new(g_get_tmp_dir());
+    g_string_append(tempfile, "/profanity-");
+    xmpp_ctx_t* const ctx = connection_get_ctx();
+    if(!ctx) {
+        cons_show("Editor: Not connection");
+        return TRUE;
+    }
+    char* uuid = xmpp_uuid_gen(ctx);
+    g_string_append(tempfile, uuid);
+    if (uuid) {
+        xmpp_free(ctx, uuid);
+    }
+    g_string_append(tempfile, ".txt");
+
+    // tempfile should be something like
+    // /tmp/profanity-f2f271dd-98c8-4118-8d47-3bd49c8e2e63.txt
+    const char* filename = tempfile->str;
+
+    // Check if file exists and create file
+    if (g_file_test(filename, G_FILE_TEST_EXISTS)) {
+        cons_show("Editor: The temp file exists");
+        g_string_free (tempfile, TRUE);
+        return TRUE;
+    }
+
+    GError* creation_error = NULL;
+    GFile* file = g_file_new_for_path(filename);
+    GFileOutputStream* fos = g_file_create (file,
+               G_FILE_CREATE_PRIVATE, NULL,
+               &creation_error);
+    if ( creation_error ) {
+        cons_show_error("Editor: Error during file creation");
+        return TRUE;
+    }
+    g_object_unref(fos);
+
+    char* editor = prefs_get_string(PREF_COMPOSE_EDITOR);
+    if (!g_file_test(editor, G_FILE_TEST_EXISTS)) {
+        cons_show_error("Editor: editor %s not exists", editor);
```

Comment:
Should be `"Editor: editor %s does not exists"`.

Reply:
or maybe "Editor: %s does not exist"

Reply:
done in e1249d9bbbe6a7a0741c8a83697f7bf962f4f82a